src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerGeneralApplicationReceived.java

```diff
@@ -16,18 +32,75 @@
 import static uk.gov.hmcts.divorce.divorcecase.model.UserRole.SUPER_USER;
 import static uk.gov.hmcts.divorce.divorcecase.model.access.Permissions.CREATE_READ_UPDATE;
 
+@Slf4j
 @Component
 public class CaseworkerGeneralApplicationReceived implements CCDConfig<CaseData, State, UserRole> {
-    public static final String CASEWORKER_GENERAL_APPLICATION_RECEIVED = "caseworker-general-application-received";
+    public static final String CASEWORKER_GENERAL_APPLICATION_RECEIVED = "cw-general-application-received";
```

Comment:
caseworker to cw ?

Reply:
Had to shorten the name due to issue with the length of event&page combination (70 currently).